OpenRA.Game/Network/UnitOrders.cs

```diff
@@ -67,6 +67,35 @@ public static void ProcessOrder(OrderManager orderManager, World world, int clie
 						break;
 					}
 
+				case "ReplacePlayerWithBot":
+					{
+						var client = orderManager.LobbyInfo.ClientWithIndex(clientId);
+						if (client != null && world != null)
+						{
+							var player = world.FindPlayerByClient(client);
+							if (player == null || player.IsBot)
+								break;
+
+							client.State = Session.ClientState.Ready;
+
+							var admin = orderManager.LobbyInfo.Clients.FirstOrDefault(c => c.IsAdmin);
```

Comment:
Don't try to expand the scope here

> Added an option to replace disconnected players with AI bots

I didn't promise to fix bots on dedicated servers, nor transfer adminship and bot controlling. Those are existing limitations, but I don't think they should be tackled here.